paper/_toc.yml

```
@@ -0,0 +1,21 @@
+# Table of Contents
+#
+# Myst will respect:
+# 1. New pages
+#      - file: relative/path/to/page
+# 2. New sections without an associated page
+#      - title: Folder Title
+#        sections: ...
+# 3. New sections with an associated page
+#      - file: relative/path/to/page
+#        sections: ...
+#
+# Note: Titles defined on pages here are not recognized.
+#
+# This spec is based on the JupyterBook table of contents.
+# Learn more at https://jupyterbook.org/customize/toc.html
+
+format: jb-book
+root: Astic_Oldenburg_PGI_article_revised_v2.tex
+chapters:
+  - file: Appendix.tex
```

The reason this shows up locally but does not show up in the preview build is platform-specific file-system case sensitivity.

On Windows/Mac, files are case-insensitive, so `Appendix.tex` here matches the `appendix.tex` file. On Ubuntu (and therefore the github action), `Appendix.tex` does not match `appendix.tex`...

Simple fix is just make the cases match:
```suggestion
  - file: appendix.tex
```